### DISCUSSION TITLE
NiceGUI Demo Web App (Traefik, FastAPI, Redis, Html, FIEF)

### DISCUSSION THREAD

**Discussion by ClaudioBsh** [zauberzeug/nicegui#2551] [Show and tell]
I did create a demo web app (https://github.com/ClaudioBsh/publicdemos/releases) - based on the discussions here: https://github.com/orgs/fief-dev/discussions/335

It uses traefik, fastapi, fief (https://www.fief.dev/), redis, html and nicegui and sure some bash/python scripts under the hood and could be a helpful starting point creating a Web App.

The demo is fully working, but currently (v0.14) there is one problem still open calling FIEF admin page - all the rest is working fine.

**Comment by rodja** [comment]
Nice! Could you also add it to the list of community projects in our wiki: https://github.com/zauberzeug/nicegui/wiki#community-projects

**Comment by ClaudioBsh** [comment]
Sure, but will wait until the demo is full working. Currently there is an issue with using FIEF administration, as soon this is solved it makes sense adding it.

**Reply by ClaudioBsh** [reply]
Did it:-)
